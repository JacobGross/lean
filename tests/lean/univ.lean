import data.num

definition id (A : Type) (a : A) := a

check id Type num

check id Type' num

check id Type.{1} num

check id _ num

check id Type.{_+1} num

check id Type.{0+1} num

check id Type Type.{1}

check id Type' Type.{1}
