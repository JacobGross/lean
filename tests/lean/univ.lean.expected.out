univ.lean:5:9: error: solution computed by the elaborator forces a universe placeholder to be a fixed value, computed sort is
  Type₁
univ.lean:7:9: error: solution computed by the elaborator forces a universe placeholder to be a fixed value, computed sort is
  Type₁
id Type₁ num : Type₁
id Type₁ num : Type₁
univ.lean:13:9: error: solution computed by the elaborator forces a universe placeholder to be a fixed value, computed sort is
  Type₁
id Type₁ num : Type₁
univ.lean:17:9: error: solution computed by the elaborator forces a universe placeholder to be a fixed value, computed sort is
  Type₂
univ.lean:19:9: error: solution computed by the elaborator forces a universe placeholder to be a fixed value, computed sort is
  Type₂
